HEC.FDA.Model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -230,31 +223,31 @@ private PairedData CreateStageFrequency()
         public List<UncertainPairedData> Compute(IProvideRandomNumbers randomProvider)
         {
             Validate();
-            List<UncertainPairedData> results = new List<UncertainPairedData>();
+            List<UncertainPairedData> results = new();
             if (ErrorLevel >= ErrorLevel.Major)
             {
                 string message = "At least one component of the stage-damage compute has a major error or worse. The compute has been aborted. Empty stage-damage functions have been returned";
-                ErrorMessage errorMessage = new ErrorMessage(message, ErrorLevel);
+                ErrorMessage errorMessage = new(message, ErrorLevel);
                 ReportMessage(this, new MessageEventArgs(errorMessage));
                 return results;
             }
             else
             {
 
-                List<string> damCats = _inventory.GetDamageCategories();
-                (List<double>, List<float[]>) wsesAtEachStructureByProfile = _hydraulicDataset.GetHydraulicDatasetInFloatsWithProbabilities(_inventory, _HydraulicParentDirectory);
+                List<string> damCats = Inventory.GetDamageCategories();
+                (List<double>, List<float[]>) wsesAtEachStructureByProfile = _hydraulicDataset.GetHydraulicDatasetInFloatsWithProbabilities(Inventory, _HydraulicParentDirectory);
 
                 //Run the compute by dam cat to simplify data collection 
                 foreach (string damageCategory in damCats)
                 {
                     //These are the stages of the stage-damage function - the aggregation stages 
-                    List<double> allStagesAtIndexLocation = new List<double>();
+                    List<double> allStagesAtIndexLocation = new();
```

Comment:
is this new syntax? intellisence recently recommended this to me, I was hesitant.